### PR TITLE
Add pan-os-python build

### PR DESCRIPTION
**Pull request by adambaumeister**
## Status
Ready

## Related Content Pull Request
None yet - planned in next 2-4 weeks

## Description
This dockerfile is simply python + the Python SDK for PAN-OS. We can use this to more rapidly build PAN-OS integrations or one-off scripts for customers in the future.

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -712,4 +712,7 @@ updates:
   - package-ecosystem: pip
     directory: /docker/py3-tools
     schedule:
-      interval: daily
\ No newline at end of file
+      interval: daily  - package-ecosystem: pip
+    directory: /docker/pan-os-python
+    schedule:
+     interval: daily
```
Comment (bakatzir) [inline .github/dependabot.yml:718]:
```suggestion
      interval: daily  
  - package-ecosystem: pip
    directory: /docker/pan-os-python
    schedule:
     interval: daily
```

**Comment by GuyAfik** [comment]
Hi @adambaumeister, can you please fix the conflicts?

**Comment by GuyAfik** [comment]
@glicht can you please review that one?

**Comment by GuyAfik** [comment]
@glicht can you please review that one?

**Review by glicht — APPROVED** [review]

**Pull request merged**